src/lib.rs

```diff
@@ -118,6 +472,8 @@ impl UTransport for ULinkZenoh {
         attributes: UAttributes,
     ) -> Result<(), UStatus> {
         // Do the validation
+        // TODO: Whether we should check URI is resolved or not
+        //       https://github.com/eclipse-uprotocol/uprotocol-spec/issues/42#issuecomment-1882273981
         if UriValidator::validate(&topic).is_err() {
```

Comment:
According to the latest discussion, it seems that ULinkZenoh should only use the Uuri in micro form without requiring any validation.